src/main/java/org/opentripplanner/transit/raptor/api/request/RaptorProfile.java

```diff
@@ -2,46 +2,42 @@
 
 
 import java.util.stream.Stream;
+import org.opentripplanner.transit.raptor.rangeraptor.standard.MinTravelDurationRoutingStrategy;
 
 /**
  * Several implementation are implemented - with different behaviour. Use the one
  * that suites your need best.
  */
 public enum RaptorProfile {
+
     /**
      * Multi criteria pareto search.
      */
     MULTI_CRITERIA("Mc"),
+
     /**
-     * Range Raptor finding the earliest arrival time, shortest travel duration and fewest transfers.
-     * The cost is not used.
+     * Used by Range Raptor finding the earliest-arrival-time, the shortest travel duration
+     * and the fewest transfers. Generalized-cost is not used.
      * <p/>
      * Computes result paths.
      */
     STANDARD("Standard"),
 
     /**
-     * Range Raptor finding the earliest arrival times - only. No paths are returned.
-     * Only one iteration is performed. This search is used internally to calculate
-     * heuristics.
-     * <p/>
-     * Computes best/min travel duration and number of transfers.
+     * Same as {@link #STANDARD}, but no paths are computed/returned.
      */
     BEST_TIME("StdBestTime"),
 
     /**
-     * Same as {@link #STANDARD}, but skip extra wait time - only board slack.
-     * <p/>
-     * Computes best/min travel duration and number of transfers.
+     * Used by Raptor to find the shortest travel duration ignoring wait-time. It also finds
```

Comment:
Maybe we should explain a bit more what ignoring wait-time means in practice. If I've not misunderstood, the search window is different compared to STANDARD.

Reply:
The search window is the same. It will just count for travel time, without taking waiting time into account.

Reply:
There is actually a check that you run the `MinTravelDurationStrategy` with one iteration, if set a searchWindow > 1m then it fails. I tried to document this.

Reply:
The `searchWindow` is a request parameter and controls the number of RangeRaptor iterations depending on the iteration size (In practice 1 minute). All 3 strategies are plugged into the RangeRaptorWorker so in theory they all can be used with a searchWindow > 0m, but as I have now added doc for here and in the JavaDoc it does not make sense for the `MinTravelDurationStrategy`, hence there is a check on it witch will throw an exception. The Raptor API is a programming API and the check prevent any programmers from using it incorrect.